jaeger-core/src/main/java/io/jaegertracing/internal/PropagationRegistry.java

```diff
@@ -22,7 +22,7 @@
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 
-class PropagationRegistry {
+public class PropagationRegistry {
```

Comment:
looking at https://github.com/jaegertracing/legacy-client-java/pull/20/files, I don't see it being constructed there. So why public?

Reply:
My mistake. Forgot to remove.